crossing-repair.lic

```diff
@@ -10,18 +10,72 @@ class CrossingRepair
   include DRCT
```

Comment:
While you're in here, do you mind to delete the `include` statements and ensure methods are property prefixed with the namespace, like `bput` => `DRC.bput`?

Reply:
Can do

Reply:
Handled.